tests/validation/tests/v3_api/test_airgap.py

```diff
@@ -441,104 +375,181 @@ def prepare_airgap_node(bastion_node, number_of_nodes):
                 ag_node.private_ip_address)
         bastion_node.execute_command(ag_node_restart_docker)
 
+        ag_node_user_own_docker = \
+            'ssh -i "{0}.pem" -o StrictHostKeyChecking=no {1}@{2} ' \
+            '"sudo chown {1}:{1} /home/{1}/.docker -R && ' \
+            'sudo chmod g+rwx "/home/{1}/.docker" -R"'.format(
+                bastion_node.ssh_key_name, AWS_USER,
+                ag_node.private_ip_address)
+        bastion_node.execute_command(ag_node_user_own_docker)
+
         print("Airgapped Instance Details:\nNAME: {}-{}\nPRIVATE IP: {}\n"
               "".format(node_name, num, ag_node.private_ip_address))
     return ag_nodes
 
 
-def prepare_private_registry_on_k3s_node(bastion_node, ag_node):
+def copy_certs_to_node(bastion_node, ag_node):
+    ag_node_copy_certs = \
+        'scp -i "{0}.pem" -o StrictHostKeyChecking=no certs/* ' \
+        '{1}@{2}:~/'.format(bastion_node.ssh_key_name, AWS_USER,
+                            ag_node.private_ip_address)
+    bastion_node.execute_command(ag_node_copy_certs)
+
+
+def trust_certs_on_node(bastion_node, ag_node):
+    ag_node_update_certs = \
+        'sudo cp domain.crt ' \
+        '/usr/local/share/ca-certificates/domain.crt && ' \
```

Comment:
I know k3s and rke2 support other linux distros than Ubuntu would this work on RHEL based linuxes?

Reply:
No this shouldn't work on other ones, but I also haven't gone through the steps to confirm what exactly to do when using this `system-default-registry` airgap method on other distros. 

Maybe we should caveat this by saying it only has been validated against Ubuntu right now and next enhancement should be to support other distros 🤔

Reply:
I'm fine if anyone else is by clarifying the support to only Ubuntu here maybe in the in the description and later in the job builder adding a note to the field or something like that?

Reply:
Added a comment above the function to describe that, and created https://github.com/rancher/qa-tasks/issues/147 for enhancing this